unabridged-works-volume-4/original/02-life-and-immortality.adoc

```diff
@@ -4042,16 +4276,23 @@ and shadowed out under the law in the old covenant.
 [.inline]
 ==== XXIV. Some Observations on 2 Pet. 3:14-16
 
-"`Wherefore, beloved, seeing that ye look for such things,
+[quote.epigraph, ,2 Peter 3:14-16]
```

Comment:
Same here, let's change this to being just a normal `quote.scripture` for the same reasons as I mention above. 👍